servo/cli.py

```diff
@@ -1231,6 +1239,23 @@ def check(
                 help="Delay duration. Requires --wait",
                 metavar="[DURATION]",
             ),
+            no_poll: Optional[bool] = typer.Option(
+                None,
+                "--no-poll",
+                help="Do not poll the Opsani API for commands",
+            ),
+            no_diagnostics: Optional[bool] = typer.Option(
+                None,
+                "--no-diagnostics",
+                help="Do not poll the Opsani API for diagnostics",
+                envvar="NO_DIAGNOSTICS",
+            ),
```

Comment:
Please replace this with a new option on `servo.configuration.BaseServoConfiguration`. Analogous shell invocation would be `SERVO_NO_DIAGNOSTICS=True servo run ....`